docs/dbot/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+Dbot is the Demisto machine learning bot which ingests information about indicators to determine if they are malicious or not. Since dbot requires a very specific dataset, we must format our data as per this article.
+
+
+**Please Note**: We are unable to use the Demisto Transformers (DT) within the DBot score context. 
```

Comment:
You can add an example what not to do
`DBotScore(val.Indicator == obj.Indicator)`